app/lib/frontend/templates/layout.dart

```diff
@@ -150,22 +150,41 @@ String _renderSearchBanner({
   }
   String secondaryTabsHtml;
   if (searchQuery?.sdk == SdkTagValue.dart) {
-    secondaryTabsHtml = _renderSecondaryTabs(
+    secondaryTabsHtml = _renderFilterTabs(
```

Comment:
Technically, SDK is a filter too, but we can get to a consistent naming after we have landed this.